Assets/Scripts/Combat/Combantants/EnemyCombatant.cs

```diff
@@ -25,7 +25,7 @@ private new void Update()
 
     private void UpdateUIPosition()
     {
-        if (displayUI)
+        if (displayUI && isAlive)
```

Comment:
Instead of checking both, leave `displayUI`, since that is the boolresponisble for checking if we should display UI or not. 

- When the Combatant dies, set `displayUI=false`
- When Combantant is alive  (initialization?), set `isAlive` to true